code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm

```diff
@@ -448,4 +448,60 @@
 	if (!modifier)
 		modifier = M.add_modifier(/datum/modifier/luminous, MODIFIER_REAGENT, src, _strength = 4, override = MODIFIER_OVERRIDE_STRENGTHEN)
 	if(isskeleton(M))
-		M.heal_organ_damage(10 * removed, 15 * removed)
\ No newline at end of file
+		M.heal_organ_damage(10 * removed, 15 * removed)
+
+/datum/reagent/liquid_fire
```

Comment:
This is probably the only one I have concerns about. Perhaps it should just ignite under most circumstances? As in, upon contact with the air, it lights ablaze. Would make it a bit more useful and fun.

Reply:
Napalm does that already.

Reply:
Then why is this even a necessary thing? "Ignites mobs on touch" seems weird.

Reply:
It could be used to set people on fire via injection or by ingestion, such as food poisoning, without causing a fire or setting fire alarms.